myapi/urls.py

```diff
@@ -6,6 +6,7 @@
 router = routers.DefaultRouter()
 router.register(r'tacos', views.TacoViewSet)
 router.register(r'restaurants', views.RestaurantViewSet)
+router.register(r'reviews', views.ReviewViewSet)
```

Comment:
Nice work adding these URLs in. Makes it so much easier to understand when they are uniform!